inv/inventory.go

```diff
@@ -25,7 +25,7 @@ import (
 
 // this inventory service interface
 type InventoryApp interface {
-	ListDevices(ctx context.Context, skip int, limit int, filters []store.Filter, sort *store.Sort, hasGroup *bool) ([]model.Device, error)
+	ListDevices(ctx context.Context, skip int, limit int, filters []store.Filter, sort *store.Sort, hasGroup *bool, groupName string) ([]model.Device, error)
```

Comment:
`ListDevices` takes 7 parameters now. Maybe it's worth refactoring this a little. How about you add a `ListQuery` struct, something like:
```
type ListQuery struct {
    skip int
    limit int
    filters []store.Filter
    sort *store.Sort // would it be possible to convert this to store.Sort?
    hasGroup bool // don't know why it's a pointer in ListDevices call, 
                  // maybe double check the logic to verify that nothing funky is done here
    group string
}
```